### ISSUE TITLE
Error at pip install "Could not find a version that satisfies the requirement yolox"

### ISSUE THREAD

**Issue by ggarber42** [drivendataorg/zamba#168]
Following the [installation guide](https://zamba.drivendata.org/docs/stable/install/) I found the following error:

```
ERROR: Could not find a version that satisfies the requirement yolox (unavailable) (from zamba) (from versions: 0.0.0)
ERROR: No matching distribution found for yolox (unavailable)
```
When I tried to follow the link of [yolox repository](https://github.com/r-b-g-b/YOLOX.git@build-requirements) I got 404. Then I believe this may be the problem. Also, I've attached an image below with the bellow.

![image](https://user-images.githubusercontent.com/26423380/144771524-9ad3266e-ce5f-4084-b988-eea372b7a4b4.png)

Thank you very much for your time!

**Comment by pjbull** [comment]
Thanks for letting us know!

@r-b-g-b did we stop using your fork now that your PR to yolox was merged? Do we need to update the requirements here to point there instead?

**Comment by r-b-g-b** [comment]
That was my mistake, I deleted the `r-b-g-b` branch when it was merged into the official YOLOX repo, but never updated it here. Now it looks like YOLOX has [reverted that change](https://github.com/Megvii-BaseDetection/YOLOX/pull/915), we'll have to follow up there to find out exactly why and if there's a workaround. In the meantime, I'll recreate the `r-b-g-b` branch so the install will continue to work.
